flaskapp/contract/form.py

```diff
@@ -0,0 +1,100 @@
+from flaskapp.contract.config import *
+
+class MultiCheckboxField(SelectMultipleField):
+    widget = ListWidget(html_tag='ul', prefix_label=False)
+    option_widget = CheckboxInput()
+
+class create_contractForm(FlaskForm):
+    # defining form fields
+    receiver = SelectField('Company')
+    conditions = MultiCheckboxField('Conditions', validators=[DataRequired()])
+    uploadfile = FileField('Share file', validators=[DataRequired()])
+
+    def getClientlist(self):
+        clientlist = []
+        for client in Client.query.all():
+            clientlist.append((str(client.id), client.name+", Org.nr.: "+str(client.id)))
+        return clientlist
+
+    def getConditions(self):
+        conditions = []
+        for condition in Conditions.query.all():
+            conditions.append((str(condition.id), condition.name))
+        return conditions
+
+    # save contract to db and as json-object and upload shared file to shared folder
+    def save(self, **kwargs):
+        new_contractID = Contract_sent.query.order_by(Contract_sent.id.desc()).first()
+        new_fileID = File.query.order_by(File.id.desc()).first()
+
+        if new_contractID is None:
```

Comment:
It is preferable to use a UUID for contracts instead of an incrementing variable.